### PR TITLE
Test and  do not add leading slash when there is one already #5473

### PR DESCRIPTION
**Pull request by nicholascus**

### CODE REVIEW

**Comment by DavertMik** [comment]
I waited a long for this PR!

File: src/Codeception/Module/REST.php [hunk 1]
```diff
@@ -577,7 +577,7 @@ protected function execute($method, $url, $parameters = [], $files = [])
         // allow full url to be requested
         if (strpos($url, '://') === false) {
             $url = $this->config['url'] . $url;
-            if ($this->config['url'] && strpos($url, '://') === false && $this->config['url'][0] !== '/') {
+            if ($this->config['url'] && strpos($url, '://') === false && strpos($url, '/') !== 0 && $this->config['url'][0] !== '/') {
```
Comment (Naktibalda) [inline src/Codeception/Module/REST.php]: Are you sure that this fix was correct?
Previous line overrides `$url` so the first byte of `$url` and `$this->config['url']` is the same.

Comment (nicholascus) [reply src/Codeception/Module/REST.php]: I'll check and redo that if required.

Comment (nicholascus) [reply src/Codeception/Module/REST.php]: https://github.com/Codeception/Codeception/pull/5511

**Pull request merged**